src/execution/__tests__/oneof-test.ts

```diff
@@ -152,13 +157,13 @@ describe('Execute: Handles OneOf Input Objects', () => {
           {
             locations: [{ column: 16, line: 2 }],
             message:
-              'Variable "$input" got invalid value { a: "abc", b: 123 }; Exactly one key must be specified for OneOf type "TestInputObject".',
+              'Variable "$input" got invalid value { a: "abc", b: 123 }; Within OneOf Input Object Type "TestInputObject", exactly one field must be specified, and the value for that field must be non-null.',
```

Comment:
This error seems to be a combination of two errors; ideally this `Within` should not have a capital letter since it's in the middle of a sentence. Alternatively; can the preceding semicolon be changed to a fullstop?

Reply:
we could definitely do either, but it would have to be in a different PR, a changing this behavior would effect multiple errors across the code base, and should not be mixed with this PR which introduces some new behavior. (We're already stretching things a teense by changing the existing errors while adding new ones, but I am Ok with that as it at least all stays within OneOf)